evap/evaluation/tests/tools.py

```diff
@@ -29,6 +36,13 @@
 )
 
 
+class SkipLiveServerTestsRunner(DiscoverRunner):
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
+        super().__init__(*args, **kwargs)
+        if not self.tags and not self.exclude_tags:
+            self.exclude_tags = {"live-server"}
```

Comment:
If someone uses the `SkipLiveServerTestsRunner` and passes `tags` or `exclude_tags` here, it doesn't skip live server tests anymore. I'd either expect us to just add "live-server" to `exclude_tags`, or at least assert that neither `tags` or `exclude_tags` was set (if we never expect this to happen). The current approach of continuing but with behavior that doesn't match the class name, to me, is just a trap we set.

Reply:
I think adding `--tags live-server` to start the live servers is reasonable. To mitigate the other stuff, I think this is most intuitive
```suggestion
        if "live-server" not in self.tags:
            self.exclude_tags.add("live-server")
```

Reply:
Maybe also a global for the `"live-server"` tag?

Reply:
> I think adding --tags live-server to start the live servers is reasonable

We're in a wonky world then, though, aren't we? Using the `SkipLiveServerTestsRunner` to run tests tagged with `live-server`... I see that this is what you currently do on CI (using ` manage.py test --tag live-server`). Is that also the invocation that we expect developers to use locally?

In the end fine with me if there is no other way to have django run specific. Maybe rename this runner then to just `EvapTestRunner`? The "add the exclude tag if it isn't explicitly included" behavior looks fine to me, then.

I don't think we need a constant for the tag, using the string is fine.